xdp-filter/xdp-filter.c

```diff
@@ -570,9 +567,23 @@ int do_port(const void *cfg, const char *pin_root_path)
 	__u8 flags = 0;
 	__u64 counter;
 	__u32 map_key;
+	unsigned int proto = opt->proto;
+	unsigned int mode = opt->mode;
```

Comment:
Please preserve reverse x-mas tree ordering of the variable declarations (like in the kernel networking code).